zerver/tests/test_external.py

```diff
@@ -189,6 +187,7 @@ def user_facing_assert_func(result: HttpResponse) -> None:
 
             self.assertNotEqual(result.status_code, 429)
 
+    @rate_limit_rule(1, 5, domain="api_by_user")
```

Comment:
The commit message on this commit is missing a few words that this `api_by_user` rule is the thing only used in one place.